src/rtapi/rtapi_pci.h

```diff
@@ -104,7 +104,6 @@ int pci_disable_device(struct pci_dev *dev);
           pci_resource_start((dev), (bar)) + 1))
 
 void __iomem *pci_ioremap_bar(struct pci_dev *pdev, int bar);
-inline void iounmap(volatile void __iomem *addr);
```

Comment:
What is the thought behind deleting this inline?  That is what the xenomai build fails on.